### ISSUE TITLE
[Material 3] Material 3 Day Night Theme Not Inheriting from System Theme

### ISSUE THREAD

**Issue by mmd44** [material-components/material-components-android#3470]
I am using this theme as my app theme:

` <style name="MyTheme" parent="Theme.Material3.DayNight">`

I am able to switch to night mode manually in code but when the device system (in phone settings) is set to night mode, my app is not switching.

Is this an expected behavior? Shouldn't the DayNight theme extend from a system theme and therefore switch automatically?

**Comment by paulfthomas** [comment]
Hi @mmd44, are your activities properly recreated? Please refer to this [documentation](https://github.com/material-components/material-components-android/blob/master/docs/theming/Dark.md) and https://developer.android.com/develop/ui/views/theming/darktheme#config-changes

**Comment by mmd44** [comment]
> Hi @mmd44, are your activities properly recreated? Please refer to this [documentation](https://github.com/material-components/material-components-android/blob/master/docs/theming/Dark.md) and https://developer.android.com/develop/ui/views/theming/darktheme#config-changes

Yeah, the issue is only with themes extending Material 3, I tried using:

`<style name="MyTheme" parent="Theme.MaterialComponents.DayNight">`

Works as expected; but this is no longer applicable in my case since I am already using many material 3 components...

**Comment by paulfthomas** [comment]
I just tested with our catalog app which uses `Theme.Material3.DayNight.NoActionBar` and it's working as expected. Something else must not be configured properly in your app.

**Comment by mmd44** [comment]
That is weird...Will double check and let you know

**Comment by mmd44** [comment]
Could be something related to build types...I tested using a release apk and it worked just fine...don't have much time to further investigate but it is weird that it is only happening on a material 3 theme...please feel free to close it if no further investigation is to be done...
